readme.md

```diff
@@ -1,72 +1,32 @@
 # Retrospect - Public GIT Repository #
-[![License](https://img.shields.io/badge/license-cc_by--nc--nd-brightgreen)](https://github.com/retrospect-addon/plugin.video.retrospect/blob/master/plugin.video.retrospect/license.txt)
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/retrospect-addon/plugin.video.retrospect)](https://github.com/retrospect-addon/plugin.video.retrospect/releases)
 [![Build Status](https://travis-ci.com/retrospect-addon/plugin.video.retrospect.svg?branch=master)](https://travis-ci.com/retrospect-addon/plugin.video.retrospect)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=retrospect-addon:plugin.video.retrospect&metric=alert_status)](https://sonarcloud.io/dashboard?id=retrospect-addon:plugin.video.retrospect)
-
+[![License](https://img.shields.io/badge/license-cc_by--nc--nd-brightgreen)](https://github.com/retrospect-addon/plugin.video.retrospect/blob/master/plugin.video.retrospect/license.txt)
+[![Python](https://img.shields.io/badge/python-2.7%20%7C%203.6-blue)](https://kodi.tv/article/attention-addon-developers-migration-python-3)
 
 This repository holds the main code for Retrospect. For more information of bug reporting please visit https://github.com/retrospect-addon/plugin.video.retrospect or https://www.rieter.net/content/.
 
```

Comment:
This repository holds the code for Retrospect. For more information on bug reporting, please visit https://github.com/retrospect-addon/plugin.video.retrospect.

Motivation besides language correction(s): Rieter.net only references Github and it is not updated. Until it is, maybe not reference it?

Reply:
Why is it not updated?

Reply:
Rieter.net isn't updated, right? A lot of outdated texts.

Reply:
I am going to move more and more to Github now less on my private website. So that is for a later thing.